src/main/asciidoc/_chapters/ops_mgt.adoc

```diff
@@ -2079,6 +2079,98 @@ Examples:
 
 ----
 
+[[slow_log_responses_from_systable]]
+==== Get Slow/Large Response Logs from System table hbase:slowlog
+
+The above section provides details about Admin APIs:
+
+* get_slowlog_responses
+* get_largelog_responses
+* clear_slowlog_responses
+
+All of the above APIs access online in-memory ring buffers from
+individual RegionServers and accumulate logs from ring buffers to display
+to end user. However, they can only be used for an ongoing slow/large RPC calls
+and user can retrieve ongoing issues with RPC logs. After RegionServer is
+restarted, all the objects held in memory of that RegionServer will be cleaned up
+and previous problematic logs are lost. What if we want to persist all these logs?
+What if we want to store them in such a manner that operator can get all historical
+records with some filters? e.g get me all large/slow RPC logs that are triggered by
+user1 and are related to region:
+cluster_test,cccccccc,1589635796466.aa45e1571d533f5ed0bb31cdccaaf9cf.
+
+If we have a system table that stores such logs in increasing (not so strictly though)
+order of time, it can definitely help operators debug some historical events
+(scan, get, put, compaction, flush etc) with detailed inputs.
+
+Config which enabled system table to be created and store all log events is
+`hbase.regionserver.slowlog.systable.enabled`.
+
+The default value for this config is `false`. If provided `true`
+(Note: `hbase.regionserver.slowlog.buffer.enabled` should also be `true`),
+HMaster will create system table `hbase:slowlog` and each RegionServer, upon
+determining any slow/large RPC logs at RpcServer level, will put them in in-memory
+ring buffer (since `hbase.regionserver.slowlog.buffer.enabled` is true) and
```

Comment:
The one inside the () are redundant I think. You can remove them and still it makese sense.